mrSerious/StackOverflow-lite
# #160358895 Build a front-end to consume the delete question end-point

#### What does this PR do?
Have a front-end that consume the delete question end-point
#### Description of Task to be completed?
Have a working front-end to consume the following endpoint.
`DELETE /api/v1/questions/:questionId`
#### How should this be manually tested?
After cloning the repo, cd into the folder and run `npm install`
Check that front-end consumes the endpoint as follows:
- Open your browser
- Navigate to the page to create a new question.
- Complete the form.
- Open the question details page.
- Locate and click the delete button.
- Your question is deleted successfully.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160358895](https://www.pivotaltracker.com/n/projects/2189147/stories/160358895)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
